Stylesheets/authentication.css

```diff
@@ -0,0 +1,57 @@
+.auth-main-container {
+    min-width: 33rem;
+    margin: auto;
+    position: absolute;
+    left: 43rem;
+    top: 16rem;
+    padding: 2rem;
+    border-radius: 4px;
+}
+
+.auth-header h3 {
+    font-size: 2rem;
+    font-weight: 2000;
+    text-align: center;
```

Comment:
Font-weight is a fixed value provided by the font providers. Check the weights you have imported and use them. I don't think 2000 font-weight exists.

Reply:
yes it doesn't exists. Thanks for pointing it out. I have changed it to 900